packages/mobile/src/components/overflow-menu-drawer/OverflowMenuDrawer.tsx

```diff
@@ -83,6 +90,13 @@ export const OverflowMenuDrawer = () => {
     }
   })
 
+  const { data: events } = useEventsByEntityId(id, {
```

Comment:
i added a hook `useRemixContest` which seems like it's doing a similar thing as your helper fn, but not searching for active. let's settle on a canonical way to do this, don't mind if it's your fn, and update all callsites to be consistent?

Reply:
maybe could just update `useRemixContest` to be `useActiveRemixContest` and move the logic there?